front/components/assistant_builder/TryAssistant.tsx

```diff
@@ -84,7 +84,7 @@ export function usePreviewAssistant({
         description: "Draft Agent",
         instructions: builderState.instructions,
         avatarUrl: builderState.avatarUrl ?? getDefaultAvatarUrlForPreview(),
-        scope: "private",
+        scope: "hidden",
```

Comment:
Should we avoid using string and use a variable so that if and when we need to update hidden someday we don't miss an occurrence? (not blocking for this PR of course)

Reply:
yes would be better , I'll do that in a followup pr